scalastyle_config.xml

```diff
@@ -0,0 +1,188 @@
+<scalastyle commentFilter="enabled">
```

Comment:
This is all not your change, rigth? Please check why appears so, looks like merge issue